app/api/subscriptions.py

```diff
@@ -0,0 +1,97 @@
+from app import app
+from flask import abort
+from flask.ext.restful import Api, Resource, reqparse, fields, marshal
+
+api = Api(app)
+
+logged_in_user_id = 1
+
+subscriptions = [
+	{
+		'user_id': 1,
+		'feed_id': 3,
+		'mergeable': True
+	},
+	{
+		'user_id': 2,
+		'feed_id': 4,
+		'mergeable': True
+	},
+	{
+		'user_id': 2,
+		'feed_id': 2,
+		'mergeable': True
+	}
+]
+
+subscription_fields = {
+    'mergeable': fields.Boolean,
+    'uri': fields.Url('subscription')
+}
+
+def user_subscriptions():
+	return (subscription for subscription in subscriptions if subscription['user_id'] == logged_in_user_id)
+
+def subscriptions_by_feed_id(feed_id):
+	return (subscription for subscription in user_subscriptions() if subscription['feed_id'] == feed_id)
+
+class SubscriptionListAPI(Resource):
+
+	def __init__(self):
+		self.reqparse = reqparse.RequestParser()
+		self.reqparse.add_argument('feed_id', type=int, required=True,
```

Comment:
feed_id or feed_url is required? What are we asking from the user with a subscribe request?